maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -413,6 +415,26 @@ class Orchestra(
         false
     }
 
+    private fun extractTextWithAICommand(command: ExtractTextWithAICommand): Boolean = runBlocking {
+        // Extract text from the screen using AI
+        if (ai == null) {
```

Comment:
nit: Although this `ai` variable was not added now, it probably deserves a better name